src/libolm_migration.rs

```diff
@@ -225,20 +224,21 @@ impl Migration {
     ///
     /// * `sessions` - An `Array` of {@link PickledSession}s to import.
     /// * `pickle_key` - The libolm pickle key that was used to pickle the olm
-    ///   session objects.
+    ///   session objects. Items inside `sessions` are going to be deleted on
+    ///   the JS part after the function returns.
```

Comment:
Not after the function returns.

```suggestion
    ///   session objects. Items inside `sessions` will be invalidated by this method.
```